api/src/main/java/io/smallrye/opentelemetry/api/OpenTelemetryConfig.java

```diff
@@ -4,6 +4,26 @@
 import java.util.Optional;
 
 public interface OpenTelemetryConfig {
+    String OTEL_BSP_MAX_EXPORT_BATCH_SIZE = "otel.bsp.max.export.batch.size";
+    String OTEL_BSP_MAX_QUEUE_SIZE = "otel.bsp.max.queue.size";
+    String OTEL_BSP_SCHEDULE_DELAY = "otel.bsp.schedule.delay";
+    String OTEL_EXPERIMENTAL_SDK_ENABLED = "otel.experimental.sdk.enabled";
+    String OTEL_EXPORTER_JAEGER_ENDPOINT = "otel.exporter.jaeger.endpoint";
+    String OTEL_EXPORTER_JAEGER_TIMEOUT = "otel.exporter.jaeger.timeout";
+    String OTEL_EXPORTER_OTLP_ENDPOINT = "otel.exporter.otlp.endpoint";
+    String OTEL_EXPORTER_OTLP_PROTOCOL = "otel.exporter.otlp.protocol";
+    String OTEL_EXPORTER_OTLP_TIMEOUT = "otel.exporter.otlp.timeout";
+    String OTEL_LOGS_EXPORTER = "otel.logs.exporter";
+    String OTEL_METRICS_EXPORTER = "otel.metrics.exporter";
+    String OTEL_SDK_DISABLED = "otel.sdk.disabled";
+    String OTEL_SERVICE_NAME = "otel.service.name";
+    String OTEL_SPAN_PROCESSOR_TYPE = "wildfly.span.processor.type";
+    String OTEL_TRACES_EXPORTER = "otel.traces.exporter";
+    String OTEL_TRACES_SAMPLER = "otel.traces.sampler";
+    String OTEL_TRACES_SAMPLER_ARG = "otel.traces.sampler.arg";
+    String PROTOCOL_GRPC = "grpc";
+    String PROTOCOL_HTTP = "http/protobuf";
```

Comment:
Are these being used?

Reply:
A few are used here, yes, and the others are used in WildFly. I can leave/move them back to WildFly, but it seemed like something other consumers might want, and this made that DRY?

Reply:
Changes reverted.